### DISCUSSION TITLE
Export Action

### DISCUSSION THREAD

**Discussion by yudisyudistira12** [filamentphp/filament#11436] [Help]
### Package

Actions

### Package Version

v.3.0

### How can we help you?

Below I have attached the code for presenceexporter and presence resource. When I export data to the data table, there are many duplicate data for names and the total attendance data is not available. How do you ensure that name data is not duplicated and total attendance data per month can be obtained?

![carbon](https://github.com/filamentphp/filament/assets/59831982/7713b74d-0bf7-4d04-9212-9b31c941d198)
![carbon (1)](https://github.com/filamentphp/filament/assets/59831982/459ba3fd-1745-4967-87b0-2fc51f13d04c)

**Comment by micraux** [comment]
Hello,

You may want to modify the export query (distinct, group by, sum, count...):

[https://filamentphp.com/docs/3.x/actions/prebuilt-actions/export#modifying-the-export-query](https://filamentphp.com/docs/3.x/actions/prebuilt-actions/export#modifying-the-export-query)
